pkg/exchange/binance/binanceapi/client.go

```diff
@@ -19,13 +21,31 @@ import (
 	"github.com/c9s/bbgo/pkg/types"
 )
 
-const defaultHTTPTimeout = time.Second * 15
+const defaultHTTPTimeout = time.Second * 2
 const RestBaseURL = "https://api.binance.com"
 const SandboxRestBaseURL = "https://testnet.binance.vision"
 const DebugRequestResponse = false
 
+var Dialer = &net.Dialer{
```

Comment:
it's not used from outside, make it private?

Reply:
done